### DISCUSSION TITLE
Proof of Competence Section

### DISCUSSION THREAD

**Discussion by 4gnle** [Developer-DAO/DAO-job-board#126] [Ideas]
We recently pushed a new section in the `create-profile` [form ](https://dao-job-board.vercel.app/create-profile)regarding Projects people have worked on (see #85). But we probably still need a section for people to add their education and experience (see https://github.com/Developer-DAO/DAO-job-board/issues/58).

With that in mind, I was thinking we could create a new portion on the project called `Proof of Competence` or something similar. This may overlap with the suggested collaboration from the `#proof-of-competence` project being worked on at the DAO. However, a little wording change may suffice. 

In this section, users will have the opportunity to add `Projects`, `Education`, and `Experience`. They will be listed out in their profiles so recruiters can take a look at them directly. 

We've already mentioned this - but wanted to consolidate ideas so we push this forward rn.

**Comment by carlomigueldy** [comment]
Also we could automatically fetch all POAPs and have it display on their profile, even completion NFTs.

**Comment by doswelleth-zz** [comment]
POAP or verification credential (signed with a signature) would be great! We can even create a color-coded badge system for hierarchy (hours, difficulty, etc).

**Comment by carlomigueldy** [comment]
Here is a related discussion worth taking a look at https://github.com/Developer-DAO/DAO-job-board/discussions/142
